components/Navigation.js

```diff
@@ -0,0 +1,108 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
+import { useState } from 'react';
+import Link from 'next/link';
+import Image from 'next/image';
+
+import profilePic from '../public/img/profile.jpg';
+
+export const Navigation = () => {
+  const [isMenuOpen, setIsMenuOpen] = useState(false);
+
+  const hamburgerMenuHandler = () => {
+    isMenuOpen ? setIsMenuOpen(false) : setIsMenuOpen(true);
+  };
+
+  const mobileMenu = () => (
+    <nav className="bg-accent-200 absolute top-[52px] md:top-[76px] lg:invisible right-0 p-3 z-50">
+      <ul className="flex flex-col">
+        <li className="divide-y-2 divide-black">
+          <Link href="/new-article">
+            <a className="divide-y-2 divide-black p-3">Nový článek</a>
+          </Link>
+        </li>
+        <li className="divide-y-2 divide-black">
+          <Link href="/profile">
+            <a className="divide-y-2 divide-black p-3">Profil</a>
+          </Link>
+        </li>
+        <li className="divide-y-2 divide-black">
+          <Link href="/logout">
+            <a className="divide-y-2 divide-black p-3">Odhlasit se</a>
+          </Link>
+        </li>
+      </ul>
+    </nav>
+  );
+
+  return (
+    <>
+      {isMenuOpen && mobileMenu()}
```

Comment:
I think that third option would be better than the second. Why not the first?
So it applies to all components - navigation, searchbar, header? Navigation will be inside the header?

Reply:
To the point 1 -  it is about import? than in the first case I will need to use `import Navigation from '../Navigation/Navigation.js';`  but the second and third it will be `import Navigation from '../Navigation'` ?